packages/flame_lint/lib/analysis_options.yaml

```diff
@@ -13,7 +13,6 @@ linter:
     - always_declare_return_types
     - always_put_control_body_on_new_line
     - always_put_required_named_parameters_first
-    - always_require_non_null_named_parameters
```

Comment:
Is this deprecated in Flutter beta?

Reply:
Yup, it's no longer in the pool of available lints